examples/src/features.js

```diff
@@ -6,6 +6,6 @@ export default {
         },
     },
     tasks: {
-        anyTask: true,
+        assignToGroup: true,
```

Comment:
there is a current check that was missing from this object section. this is swapping the name to add that in the example